yomigeek/phone-gen-api
# Application API Setup

**What does this Pr do?**
Application setup and generate numbers

**Description of tasks to be completed**
- add initial packages
- add the entry route
- add generate numbers route

**How should this be manually tested**
Clone the repo: Run git clone https://github.com/yomigeek/phone-gen-api.git in the folder where you want the repo to be saved.
Navigate into the newly created folder and install the dependencies: cd folder_name && npm install
Start the server using npm start
Make a GET request to http://localhost:5000/api/v1/ to ensure the API is running perfectly
Make a POST to http://localhost:5000/api/v1/request/generate with quantity as part of the request body to generate list of numbers

Screenshots (if appropriate)
N/A
